packages/endpoints-organizations-registrar/test/helpers/email-matchers.js

```diff
@@ -223,120 +220,19 @@ const expectedServiceActivationRequiredEmail = {
-                } has registered your organization to the Velocity Network to enable expanded functionality. To learn more about Velocity Network, please visit 
-    <a href="https://www.velocitynetwork.foundation/" target="_blank">https://www.velocitynetwork.foundation/</a>.
-</p>
-<p>${
-                    organization.profile.name
-                } has been assigned the following role(s) on Velocity Network:</p>
-<ul>
-
-${flow(
-                    map(
-                        (service) => `
-<li>
-  <span
-    style="
-      border: none;
-      cursor: pointer;
-    "
-  >
-    ${ServiceTypeLabels[service.type]}
-  </span>
-</li>`
-                    ),
-                    join('')
-                )(organization.services)}
-</ul>
-<br />
-<p>Please confirm your approval of the following:</p>
-<ul>
-  <li>You are the authorized signatory for your organization.</li>
-  <li>${organization.profile.adminGivenName} ${
-                    organization.profile.adminFamilyName
-                } has been nominated as the administrator of ${
-                    organization.profile.name
-                } on the Velocity Network.</li>
-  <li>${organization.profile.adminGivenName} ${
-                    organization.profile.adminFamilyName
-                } is authorized to accept the applicable 
-  <a href="https://www.velocitynetwork.foundation/main2/participation-agreements" target="_blank">terms of participation</a>.</li>
-</ul>
-<div style="display: inline-block">
-  <a
-    href="https://ui.example.com/signatories/approve?authCode=1&did=${organization.didDoc.id}"
-    target="_blank"
-    style="
-      border: none;
-      color: white;
-      background-color: #0277ff;
-      text-decoration: none;
-      padding: 5px 15px;
-      text-align: center;
-      display: inline-block;
-      font-size: 16px;
-      margin: 4px 2px;
-      cursor: pointer;
-    "
-  >Approve</a>
-  <span>&nbsp;</span>
-  <a
-    href="https://ui.example.com/signatories/reject?authCode=1&did=${organization.didDoc.id}"
-    target="_blank"
-    style="
-      border: none;
-      color: white;
-      background-color: #0277ff;
-      text-decoration: none;
-      padding: 5px 15px;
-      text-align: center;
-      display: inline-block;
-      font-size: 16px;
-      margin: 4px 2px;
-      cursor: pointer;
-    "
-  >Reject</a>
-</div>
-<br />
-<p>
-    <strong>If the buttons above do not work, please copy and paste the appropriate link into your browser:</strong>
-</p>
-<p>
-    <strong>Approve:</strong>
-<br />
-<span>https://ui.example.com/signatories/approve?authCode=1&did=${
-                    organization.didDoc.id
-                }</span>
-</p>
-
-<p>
-    <strong>Reject:</strong>
-    <br />
-    <span>
-        https://ui.example.com/signatories/reject?authCode=1&did=${organization.didDoc.id}
-    </span>
-</p>
-<br />
-
-`,
-                /* eslint-enable */
-      },
+      Text: { Data: messageMatcher },
```

Comment:
Add a default messsageMatcher verify some core introduction of the message and buttons that should be there